### DISCUSSION TITLE
How to generate bot string session ?

### DISCUSSION THREAD

**Discussion by WeebZoneIndia** [aahnik/tgcf#64] [Q&A] [answered]
how to generate a bot string session?

**Comment by aahnik** [comment]
For now, `tgcf` is not yet optimized to run on the cloud.

You can run `tgcf` on your computer. when You run tgcf for the first time, it will ask you for your phone no. (will then required otp) or bot token. after you log in, it will save the session files in the current directory (the directory from which you ran `tgcf` command)

for generating session string, this feature will be added by tomorrow. I am working on making an easy experience of deploying to cloud.
Star and watch this repo to get notification of a new release.

the deploy buttons are not yet properly configured. it will be available by apr 22.

**Comment by aahnik** [comment, marked as answer]
hi @WeebTime 

Read this How to [Login with a bot or user account](https://github.com/aahnik/tgcf/wiki/Login-with-a-bot-or-user-account) 

and for generating session string on Heroku, read this:  [One click deploy](https://github.com/aahnik/tgcf/wiki/Deploy-to-Heroku#one-click-deploy)

**Comment by WeebZoneIndia** [comment]
ok
